hydra-node/test/Hydra/Model.hs

```diff
@@ -97,6 +97,7 @@ data GlobalState
   | Idle
       { idleParties :: [Party]
       , cardanoKeys :: [VerificationKey PaymentKey]
+      , idleContestationPeriod :: ContestationPeriod
```

Comment:
why?
is this part of the `WorldState`?

Reply:
It made sense since now we configure our hydra-node on start and rely on this value in the rest of the code. `Idle` is similar in a sense you need to provide values like parties, keys etc. before trying to run the tests.